[2023-09-05T11:35:47.966+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-09-04T00:00:00+00:00 [queued]>
[2023-09-05T11:35:47.977+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-09-04T00:00:00+00:00 [queued]>
[2023-09-05T11:35:47.977+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-05T11:35:47.977+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-05T11:35:47.978+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-05T11:35:47.989+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-09-04 00:00:00+00:00
[2023-09-05T11:35:47.995+0000] {standard_task_runner.py:55} INFO - Started process 87 to run task
[2023-09-05T11:35:48.000+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-09-04T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpg4_3erwx']
[2023-09-05T11:35:48.002+0000] {standard_task_runner.py:83} INFO - Job 90: Subtask create_table
[2023-09-05T11:35:48.113+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-09-04T00:00:00+00:00 [running]> on host 993b415b7ab0
[2023-09-05T11:35:48.276+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-09-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-04T00:00:00+00:00
[2023-09-05T11:35:48.309+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-09-05T11:35:48.315+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-09-05T11:35:48.316+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-09-05T11:35:48.325+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230904T000000, start_date=20230905T113547, end_date=20230905T113548
[2023-09-05T11:35:48.388+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-05T11:35:48.450+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
